AoLibs.Adapters.Android/FileStorageProvider.cs

```diff
@@ -127,6 +127,12 @@ public void RemoveFile(string path)
             file.Delete();
         }
 
+        public bool CheckIfFileExist(string path)
+        {
+            var file = new File(ResolvePath(path));
+            return file.ParentFile.Exists();
```

Comment:
Why `file.ParentFile.Exists();` and not `file.Exists();`?

Reply:
Updated